src/modules/CRUD/components/Nav/NavIcon/NavIcon.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Icon } from '@blueprintjs/core';
+
+const NavIcon = ({ src, alt = '', isGroup, className, ...rest }) =>
+  (src
+    ? (
+      <span {...rest} className={`${className} bp3-icon`}>
+        <img className="CRUD-nav__icon" src={src} alt={alt} />
+      </span>
+    )
+    : <Icon {...rest} icon={isGroup ? 'layers' : 'layer'} color="#333" />
```

Comment:
```suggestion
    : <Icon {...rest} className={className} icon={isGroup ? 'layers' : 'layer'} color="#333" />
```

Reply:
I'm not sure for this one the `Icon` component has already its own classNames, i don't want to override them

Reply:
You won't override anything: BlueprintJS will merge its own classes and those you provides : https://github.com/palantir/blueprint/blob/1aa71605/packages/core/src/components/icon/icon.tsx#L116

Reply:
Ok, top!